extensions/binding/org.eclipse.smarthome.binding.onewire/ESH-INF/thing/common.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onewire" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0" xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<!-- Device Present Channel -->
+	<channel-type id="present">
+		<item-type>Switch</item-type>
+		<label>Present</label>
+		<description>ON if device present on OneWire bus</description>
+		<state readOnly="true" />
+	</channel-type>
+	<!-- Temperature Channel -->
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>temperature value of this sensor</description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+		<config-description>
+			<parameter name="resolution" type="text">
+				<label>Sensor resolution</label>
+				<options>
+					<option value="9">9 bit</option>
+					<option value="10">10 bit</option>
+					<option value="11">11 bit</option>
+					<option value="12">12 bit</option>
+				</options>
+				<default>10</default>
+				<limitToOptions>true</limitToOptions>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</channel-type>
+	<!-- Dewpoint Channel -->
+	<channel-type id="dewpoint">
+		<item-type>Number:Temperature</item-type>
+		<label>Dewpoint</label>
+		<description>dewpoint (calculated from temperature and relative humidity)</description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<!-- Relative Humidity Channel -->
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>relative humidity (0-100%)</description>
+		<state readOnly="true" pattern="%d %%" />
+	</channel-type>
+	<channel-type id="humidityconf">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>relative humidity (0-100%)</description>
+		<state readOnly="true" pattern="%d %%" />
+		<config-description>
+			<parameter name="humiditytype" type="text">
+				<label>Humidity Sensor-Type</label>
+				<limitToOptions>true</limitToOptions>
+				<options>
+					<option value="/humidity">HIH-3610</option>
+					<option value="/HIH4000/humidity">HIH-4000</option>
+					<option value="/HTM1735/humidity">HTM-1735</option>
+					<option value="/DATANAB/humidity">Datanab</option>
+				</options>
+				<default>/humidity</default>
+				<required>true</required>
```

Comment:
Agreed on the "required" setting. But I think it is more transparent to the iser if he can select the sensor he has rather than leaving the selection empty (especially when configuiring via Paper UI).

Reply:
The Paper UI should actually pro-actively fill the field with the provided default, so the selection should not be empty (otherwise I'd consider it a bug in the Paper UI, which you might report).